aixplain/modules/model/__init__.py

```diff
@@ -208,17 +209,28 @@ def run(
             response = self.run_async(data, name=name, parameters=parameters)
             if response["status"] == "FAILED":
                 end = time.time()
-                response["elapsed_time"] = end - start
-                return response
+                return ModelResponse(
+                    status="FAILED",
```

Comment:
Create an enumerator for the status with the values `FAILED`, `IN_PROGRESS` and `SUCCESS`. As a reference, follow this example: https://github.com/aixplain/aiXplain/blob/main/aixplain/enums/asset_status.py